src/main/js/page_objects/blueocean/bluePipelineBranch.js

```diff
@@ -0,0 +1,30 @@
+var url = require('../../util/url');
+
+module.exports = {
+    elements: {
+        runButton: 'div.actions div.run-pipeline',
+        toasts: 'div.toaster div.toast span.text'
+    }
+};
+
+// Nightwatch commands.
+// http://nightwatchjs.org/guide#writing-commands
+module.exports.commands = [{
+    clickRunButton: function (browser) {
+        var self = this;
+        self.waitForElementVisible('@runButton');
+        self.click('@runButton');
+        browser.elements('css selector', 'div.toaster div.toast span.text', function (codeCollection) {
```

Comment:
Why re you doing this. And you have already defined a selector for `div.toaster div.toast span.text` in the page object `elements`.